DelegateFun/Sorter.Tests/SortUtilityTests.cs

```diff
@@ -1,13 +1,66 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using static Sorter.SortUtility;
 
 namespace Sorter.Tests
 {
     [TestClass]
     public class SortUtilityTests
     {
+
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void SortUtilNullArray_Fail()

```

Comment:
You may want the name of your test(s) to consist of three parts:

The name of the method being tested.
The scenario under which it's being tested.
The expected behavior when the scenario is invoked.

For more info i'd suggest [Microsoft's Unit testing best practices](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices), really helped me out when I was confused about this
```suggestion
        public void Sort_NullArray_ThrowsException()
```